lib/linearize.js

```diff
@@ -658,7 +659,7 @@ class LinearizedCoreSession extends EventEmitter {
     this._activeRequests = []
     this._checkout = opts.checkout || null
     this._clock = opts.clock || null
-    this._unwrap = opts.unwrap === true
+    this._unwrap = (opts.unwrap === true || core._unwrap === true)
```

Comment:
What if `{ unwrap: false }`? It would still unwrap if parent unwraps

Reply:
Which seems needed for this:
https://github.com/holepunchto/autobase/blob/1c3833d27e4c3f810351f266033a9b7f70a1245d/lib/linearize.js#L728

Reply:
Ah you're right, thanks! Fixed now with:

```
diff --git a/lib/linearize.js b/lib/linearize.js
index 8cc9e3e..097ce77 100644
--- a/lib/linearize.js
+++ b/lib/linearize.js
@@ -659,7 +659,7 @@ class LinearizedCoreSession extends EventEmitter {
     this._activeRequests = []
     this._checkout = opts.checkout || null
     this._clock = opts.clock || null
-    this._unwrap = (opts.unwrap === true || core._unwrap === true)
+    this._unwrap = opts.unwrap === true ?? core._unwrap === true
     this._pinned = opts.pin === true
     this._snapshotted = opts.snapshot === true
     this._sessionIdx = -1 // Set in BranchSnapshot.session

```